frontend/src/metabase/admin/permissions/constants/collections-permissions.js

```diff
@@ -0,0 +1,23 @@
+import { t } from "ttag";
+import { color } from "metabase/lib/colors";
+
+export const COLLECTION_OPTIONS = {
+  write: {
+    label: t`Curate`,
+    value: "write",
+    icon: "check",
+    iconColor: color("success"),
```

Comment:
mentioned in another comment, but I think whitelabeling might now work with this `color` calls that run on instantiation